tools/ci/log4j.properties

```diff
@@ -37,15 +37,33 @@ appender.file.fileName = ${sys:log.dir}/mvn-${sys:mvn.forkNumber}.log
 appender.file.layout.type = PatternLayout
 appender.file.layout.pattern = %d{HH:mm:ss,SSS} [%20t] %-5p %-60c %x - %m%n
 
+# -----------------------------------------------------------------------------
+# ZooKeeper-related logs should end up here
+# -----------------------------------------------------------------------------
+appender.zk.name = ZooKeeperAppender
+appender.zk.type = FILE
+appender.zk.fileName = ${sys:log.dir}/zookeeper-${sys:mvn.forkNumber}.log
+appender.zk.layout.type = PatternLayout
+appender.zk.layout.pattern = %d{HH:mm:ss,SSS} [%20t] %-5p %-60c %x - %m%n
+
 # suppress the irrelevant (wrong) warnings from the netty channel handler
 logger.netty.name = org.jboss.netty.channel.DefaultChannelPipeline
 logger.netty.level = ERROR
+
+# collecting ZooKeeper-related logs in separate log file
 logger.zookeeper.name = org.apache.zookeeper
-logger.zookeeper.level = ERROR
+logger.zookeeper.level = INFO
+logger.zookeeper.additivity = false
+logger.zookeeper.appenderRef.zk.ref = ZooKeeperAppender
 logger.shaded_zookeeper.name = org.apache.flink.shaded.zookeeper3
-logger.shaded_zookeeper.level = ERROR
+logger.shaded_zookeeper.level = INFO
+logger.shaded_zookeeper.additivity = false
+logger.shaded_zookeeper.appenderRef.zk.ref = ZooKeeperAppender
 logger.zkquorum.name = org.apache.zookeeper.server.quorum.QuorumCnxManager
```

Comment:
this have no effect as it's already covered by the `logger.zookeeper`

Reply:
Good point 👍